### ISSUE TITLE
🐛 Bug Report: userInfo graphql enpoint not work

### ISSUE THREAD

**Issue by tupe12334** [amplication/amplication#2276]
### What happened?

The userInfo endpoint in the auth resolver is not working due to that the return object is UserInfo and UserInfo require token.
In the userInfo endpoint we are not regenerate the token so we need to update the UserInfo object to allow nulleble token

### What you expected to happen

Update the UserInfo object
```@ObjectType()
export class UserInfo implements Partial<User> {
  @Field(() => String)
  username!: string;
  @Field(() => [String])
  roles!: string[];
  @Field(() => String, { nullable: true })
  accessToken?: string;
}
```

### How to reproduce

query the userInfo endpoint

### Amplication version

0.11.2

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by mshidlov** [comment]
@tupe12334 thank you for reporting this issue, can you please give a description of the specific service? is this bug related to @amplication/server? generated app code? @amplication/design-system?

The "what happened?" and "how to reproduce?" needs a little more explanation on the location of the endpoint.

**Comment by tupe12334** [comment]
resolved in #2041
